spanner_orm/admin/column.py

```diff
@@ -39,8 +41,36 @@ def nullable(self) -> bool:
 
   def field_type(self) -> Type[field.FieldType]:
     for field_type in field.ALL_TYPES:
-      if self.spanner_type == field_type.ddl():
```

Comment:
I think this is not the good way to check if type matches. The `ddl()` method should probably go away altogether as it is not scaling at this point. Instead, you should add `field_type.matches(spanner_type)`. The `matches()` function can safely check if the type is matching and then ignore the length.

Reply:
Done. Added matches() in _type.